### PR TITLE
Cf 16 Organizing Header

### PR DESCRIPTION
**Pull request by mrabiabrn**
Proposed changes
----------------

* Separated Styles for header.
* Deleted Settings & Navigation buttons.
* Added Router for Profile page

Types of changes
----------------

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

### CODE REVIEW

**Review by erencanuysal — APPROVED** [review]

**Review by erencanuysal — APPROVED** [review]

File: Columbus/frontend/src/pages/Profile/Profile.js [hunk 1]
```diff
@@ -117,137 +76,6 @@ function Profile(props) {
...
-              </Container>
-            </div>
-          </Container>
-          <CreatePostDialog></CreatePostDialog>
-          {postIds.length === 0 && renderEmptyPost()}
-          {postIds.length !== 0 &&
-            postIds.map((item, index) => {
-              console.log("item: ", item.id);
-              return <PostCard key={index} props={{ id: 3 }}></PostCard>;
-            })}
```
Comment (CodeBlo) [inline Columbus/frontend/src/pages/Profile/Profile.js:250]: Should this be deleted? Isn't this the whole profile page?

**Review by CodeBlo — CHANGES_REQUESTED** [review]

File: Columbus/frontend/src/pages/Profile/Profile.js (hunk 1 above)
Comment (mrabiabrn) [reply Columbus/frontend/src/pages/Profile/Profile.js:250]: This was the practice app’s profile page and it was giving sone import errors. So I cleaned up the Profile page in this branch. We will merge the new version in Cf-9

Comment (CodeBlo) [reply Columbus/frontend/src/pages/Profile/Profile.js:250]: This might create conflicts. But it look goods.

**Review by CodeBlo — APPROVED** [review]

**Review by onurcanavci — APPROVED** [review]
Perfect 🤩

**Pull request merged**